src/www/ui_tests/api/Models/UploadTest.php

```diff
@@ -30,12 +30,12 @@ public function testDataFormat()
     $hash = new Hash('sha1checksum', 'md5checksum', 'sha256checksum', 123123);
     $upload = new Upload(2, 'root', 3, '', 'my.tar.gz', '01-01-2020', 3, $hash);
     $expectedUpload = [
-      "folderid"    => 2,
-      "foldername"  => 'root',
+      "folderId"    => 2,
+      "folderName"  => 'root',
       "id"          => 3,
       "description" => '',
-      "uploadname"  => 'my.tar.gz',
-      "uploaddate"  => '01-01-2020',
+      "uploadName"  => 'my.tar.gz',
+      "uploadDate"  => '01-01-2020',
       "assignee"    => 3,
       "hash"        => $hash->getArray()
```

Comment:
Please make the tests version aware. See https://github.com/fossology/fossology/pull/2617/files#diff-97059fc9a26e02adf4caa85aa5714fe79354dd2c7a33b9d5b303e8be10404834 for reference.